### PR TITLE
Applied formatting and punctuation cleanup to code comments 

### PR DESCRIPTION
**Pull request by WilliamBerryiii**
This PR is a series of code comment clean-ups, formatting and normalization based on a linear read through the compiler project.  I'm not confident that this type of work is appreciated and I have only addressed the comments for the first 6 folders of the project (what I have read to date).  Most of the changes are around the summary comments so that context box text in VS appears more consistent.  If this is work that is appreciated, I am happy to continue.  Lastly, this is done in a volume size to make the review time more efficient.

### CODE REVIEW

**Comment by KevinRansom** [comment]
@WilliamBerryiii ordinarily, I would welcome this work.  However, right at the moment we are trying to rationalize the FSHarp.Compiler Service fork with, you may have noticed a number of PR's with realign FCS in the title.  And so we probably need to keep the file delta's to a minimum until that is done.

So I would suggest holding off on this work for a few weeks would probably be a good plan.  I will ask @dsyme to let us know when the FCS alignment is complete, so we can restart this piece of work.

Thanks for making a start on this, I know it will take an age to get through the codebase.

Kevin
`

File: src/absil/il.fs [hunk 1]
```diff
@@ -165,7 +165,7 @@ let splitTypeNameRight nm =
 
 /// This is used to store event, property and field maps.
 ///
-/// Review: this is not such a great data structure.
+/// REVIEW: this is not such a great data structure.
```
Comment (KevinRansom) [inline src/absil/il.fs]: This is not an actionable review comment.  With no detail why it is "not such a great data structure"  I would say lose the comment.  In principle, Review: should probably point out a future course of action.

File: src/absil/il.fs [hunk 2]
```diff
@@ -2108,7 +2106,7 @@ let exitsOfCode c = CodeLabels.toList (exitsOfCodeAsSet c)
 /// Finds all labels defined within this code block, seeing through restrictions.
 /// This assumes that labels are unique within the code blocks, even if hidden behind restrictions.
 ///
-// Note: Repeats in the list indicate this invariant is broken.
+// NOTE: Repeats in the list indicate this invariant is broken.
```
Comment (KevinRansom) [inline src/absil/il.fs]: It's just a comment the NOTE: is probably redundent.

File: src/absil/il.fs [hunk 3]
```diff
@@ -4022,7 +4019,7 @@ let buildILCode methName lab2pc instrs tryspecs localspecs =
             try 
                 doStructure (specs1 @ specs2) (bblocks,bbstartToCodeLabelMap.BasicBlockStartCodeLabels) 
             with :? KeyNotFoundException->
-                // NOTE#overlap.
+                // NOTE: #overlap.
```
Comment (KevinRansom) [inline src/absil/il.fs]: NOTE: redundant?

File: src/absil/il.fs [hunk 4]
```diff
@@ -4501,7 +4497,7 @@ type ILGlobals with
 
     member this.mkCompilerGeneratedAttribute () = mkILCustomAttribute this (tref_CompilerGeneratedAttribute this, [], [], [])
 
-// Bug 2129. Requests attributes to be added to compiler generated methods 
+// Bug 2129. Requests attributes to be added to compiler generated methods.
```
Comment (KevinRansom) [inline src/absil/il.fs]: BUG 2129:   refers to an internal Microsoft Bug DB, and given how low the number is it is a really old bug.   We can lose the bug reference, the comment is fine though.

File: src/absil/il.fsi [hunk 5]
```diff
@@ -286,18 +286,18 @@ type ILGenericVariance =
 /// Type refs, i.e. references to types in some .NET assembly
 [<Sealed>]
 type ILTypeRef =
-    /// Create a ILTypeRef
+    /// Create a ILTypeRef.
```
Comment (KevinRansom) [inline src/absil/il.fsi]: an ?

File: src/absil/il.fsi [hunk 6]
```diff
@@ -286,18 +286,18 @@ type ILGenericVariance =
...
     static member Create : scope: ILScopeRef * enclosing: string list * name: string -> ILTypeRef
 
     /// Where is the type, i.e. is it in this module, in another module in this assembly or in another assembly? 
     member Scope: ILScopeRef
     /// The list of enclosing type names for a nested type. If non-nil then the first of these also contains the namespace.
     member Enclosing: string list
-    /// The name of the type. This also contains the namespace if Enclosing is empty 
+    /// The name of the type. This also contains the namespace if Enclosing is empty .
```
Comment (KevinRansom) [inline src/absil/il.fsi]: empty .   probably lose the space before the punctuation.

File: src/absil/il.fsi [hunk 7]
```diff
@@ -1144,7 +1143,7 @@ type ILOverridesSpec =
     member MethodRef: ILMethodRef
     member EnclosingType: ILType 
 
-// REVIEW: fold this into ILMethodDef
+// REVIEW: fold this into ILMethodDef.
```
Comment (KevinRansom) [inline src/absil/il.fsi]: Can we add an issue to do this work?, then add the github issue number.  If we close the issue without doing the work, the issue should note that these comments should be updated.

How does that sound?

File: src/absil/il.fsi [hunk 8]
```diff
@@ -1178,22 +1177,22 @@ type MethodCodeKind =
 /// may include the bounds, if any, on the generic parameter.
 type ILGenericParameterDef =
     { Name: string;
-    /// At most one is the parent type, the others are interface types 
+    /// At most one is the parent type, the others are interface types.
```
Comment (KevinRansom) [inline src/absil/il.fsi]: is indent correct?

File: src/absil/il.fsi [hunk 9]
```diff
@@ -1219,7 +1218,7 @@ type ILMethodDef =
       IsManaged: bool;
       IsForwardRef: bool;
       SecurityDecls: ILPermissions;
-      /// Note: some methods are marked "HasSecurity" even if there are no permissions attached, e.g. if they use SuppressUnmanagedCodeSecurityAttribute 
+      /// NOTE: some methods are marked "HasSecurity" even if there are no permissions attached, e.g. if they use SuppressUnmanagedCodeSecurityAttribute
```
Comment (KevinRansom) [inline src/absil/il.fsi]: Again with the NOTE: on a comment :-)

File: src/absil/il.fsi [hunk 10]
```diff
@@ -1453,7 +1458,8 @@ and [<NoComparison; NoEquality>]
       Extends: ILType option; 
       Methods: ILMethodDefs;
       SecurityDecls: ILPermissions;
-    /// Note: some classes are marked "HasSecurity" even if there are no permissions attached, e.g. if they use SuppressUnmanagedCodeSecurityAttribute 
+      /// NOTE: some classes are marked "HasSecurity" even if there are no permissions attached,
```
Comment (KevinRansom) [inline src/absil/il.fsi]: Ditto NOTE:

File: src/absil/il.fsi [hunk 11]
```diff
@@ -1562,9 +1568,9 @@ type ILAssemblyManifest =
     { Name: string;
       /// This is ID of the algorithm used for the hashes of auxiliary
```
Comment (KevinRansom) [inline src/absil/il.fsi]: This is the ID ?

File: src/absil/il.fsi [hunk 12]
```diff
@@ -1690,8 +1696,9 @@ type IPrimaryAssemblyTraits =
 // ==================================================================== *)
 
 /// A table of common references to items in primary assebly (System.Runtime or mscorlib).
-/// If you have already loaded a particular version of system runtime assembly you should reference items via an ILGlobals for that particular 
-/// version of system runtime assembly built using mkILGlobals. 
+/// If you have already loaded a particular version of system runtime assembly you should
```
Comment (KevinRansom) [inline src/absil/il.fsi]: If a particular version of System.Runtime.dll has been loaded then

File: src/absil/il.fsi [hunk 13]
```diff
@@ -1690,8 +1696,9 @@ type IPrimaryAssemblyTraits =
 // ==================================================================== *)
 
 /// A table of common references to items in primary assebly (System.Runtime or mscorlib).
-/// If you have already loaded a particular version of system runtime assembly you should reference items via an ILGlobals for that particular 
-/// version of system runtime assembly built using mkILGlobals. 
+/// If you have already loaded a particular version of system runtime assembly you should 
+/// reference items via an ILGlobals for that particular version of system runtime assembly
```
Comment (KevinRansom) [inline src/absil/il.fsi]: reference items from it via an ILGlobals for that specific version built using mkILGlobals.

File: src/absil/il.fsi [hunk 14]
```diff
@@ -1791,11 +1798,11 @@ val mkMscorlibBasedTraits : ILScopeRef -> IPrimaryAssemblyTraits
...
 
 /// When writing a binary the fake "toplevel" type definition (called <Module>)
-/// must come first. This function puts it first, and creates it in the returned list as an empty typedef if it 
-/// doesn't already exist.
+/// must come first. This function puts it first, and creates it in the returned 
+/// list as an empty typedef if it doesn't already exist.
 val destTypeDefsWithGlobalFunctionsFirst: ILGlobals -> ILTypeDefs -> ILTypeDef list
 
-/// Note: not all custom attribute data can be decoded without binding types.  In particular 
+/// NOTE: not all custom attribute data can be decoded without binding types.  In particular
```
Comment (KevinRansom) [inline src/absil/il.fsi]: Ditto NOTE:

File: src/absil/illib.fs [hunk 15]
```diff
@@ -340,7 +340,7 @@ module List =
           | _ -> finished <- true
         List.rev r, s
 
-    // note: not tail recursive 
+    // NOTE: not tail recursive
```
Comment (KevinRansom) [inline src/absil/illib.fs]: Ditto Note:  Last one I'm going to mention, if you see one in a comment it is redundant by definition :-)

File: src/absil/ilmorph.fs [hunk 16]
```diff
@@ -17,7 +17,7 @@ type EnclosingTypeDefs = ILTypeDef list * ILTypeDef
 let checking = false 
 let notlazy v = Lazy.CreateFromValue v
 
-// REVIEW: Consider removing Post-Dev11 M3
+// REVIEW: Consider removing Post-Dev11 M3.
```
Comment (KevinRansom) [inline src/absil/ilmorph.fs]: We are a long way post Dev11 M3 this comment can disapear

File: src/absil/ilmorph.fsi [hunk 17]
```diff
@@ -44,6 +44,6 @@ type InstrMorph =
 
 val morphExpandILInstrsInILCode: (ILCodeLabel -> ILCodeLabel -> ILInstr -> InstrMorph) -> ILCode -> ILCode
 
-// REVIEW: Consider removing Post-Dev11 M3
+// REVIEW: Consider removing Post-Dev11 M3.
```
Comment (KevinRansom) [inline src/absil/ilmorph.fsi]: Ditto post dev11 M3

File: src/absil/ilprint.fs [hunk 18]
```diff
@@ -529,7 +528,7 @@ let rec goutput_apps env os =  function
       output_string os "--> "; 
       goutput_typ env os ty
 
-/// utilities to help print out short forms of instructions
+/// Utilities to help print out short forms of instructions.
```
Comment (KevinRansom) [inline src/absil/ilprint.fs]: Print the short forms of instructions.

File: src/absil/ilread.fs [hunk 19]
```diff
@@ -105,7 +105,7 @@ type BinaryFile() =
     abstract CountUtf8String : addr:int -> int
     abstract ReadUTF8String : addr: int -> string
 
-/// Read file from memory mapped files
+/// Read file from memory mapped files.
```
Comment (KevinRansom) [inline src/absil/ilread.fs]: Read from memory mapped files.

File: src/absil/ilread.fs [hunk 20]
```diff
@@ -3168,7 +3168,7 @@ and sigptrGetILNativeType ctxt bytes sigptr =
            if (u = int nt_MAX) then 
              ILNativeType.Empty, sigptr'
            else
-           (* note: go back to start and read native type *)
+           (* NOTE: go back to start and read native type *)
```
Comment (KevinRansom) [inline src/absil/ilread.fs]: Surely a single line comment works here?

File: src/absil/ilreflect.fs [hunk 21]
```diff
@@ -310,13 +310,13 @@ let convAssemblyRef (aref:ILAssemblyRef) =
...
-/// The global environment
+/// The global environment.
 type cenv = 
     { ilg: ILGlobals; 
       generatePdb: bool;
       resolvePath: (ILAssemblyRef -> Choice<string,System.Reflection.Assembly> option) }
 
-/// Convert an Abstract IL type reference to Reflection.Emit System.Type value
+/// Convert an Abstract IL type reference to Reflection.Emit System.Type value.
 // REVIEW: This ought to be an adequate substitute for this whole function, but it needs
```
Comment (KevinRansom) [inline src/absil/ilreflect.fs]: Add an issue for this review item.

File: src/absil/ilreflect.fs [hunk 22]
```diff
@@ -522,7 +522,7 @@ and convTypeAux cenv emEnv preferCreated typ =
     | ILType.Byref eltType      -> let baseT = convTypeAux cenv emEnv preferCreated eltType |> nonNull "convType: byref eltType"
                                    baseT.MakeByRefType()                               |> nonNull "convType: byref" 
     | ILType.TypeVar tv         -> envGetTyvar emEnv tv                                |> nonNull "convType: tyvar" 
-  // XXX: REVIEW: complete the following cases.                                                        
+    // XXX: REVIEW: complete the following cases.
```
Comment (KevinRansom) [inline src/absil/ilreflect.fs]: Add an issue, lose the XXX:

File: src/absil/ilreflect.fs [hunk 23]
```diff
@@ -558,7 +558,7 @@ let convTypes cenv emEnv (typs:ILTypes) = ILList.map (convType cenv emEnv) typs
 
 let convTypesToArray cenv emEnv (typs:ILTypes) = convTypes cenv emEnv typs |> ILList.toArray 
 
-/// Uses the .CreateType() for emitted type (if available)
+/// Uses the .CreateType() for emitted type (if available).
```
Comment (KevinRansom) [inline src/absil/ilreflect.fs]: parens not needed for if available.

File: src/absil/ilreflect.fs [hunk 24]
```diff
@@ -1388,10 +1388,10 @@ let convMethodImplFlags mdef =
 //----------------------------------------------------------------------------
   
 let rec buildMethodPass2 cenv tref (typB:TypeBuilder) emEnv (mdef : ILMethodDef) =
-   // remaining REVIEW:
-   // SecurityDecls: Permissions;
-   // IsUnmanagedExport: bool; (* -- The method is exported to unmanaged code using COM interop. *)
-   // IsMustRun: bool; (* Whidbey feature: SafeHandle finalizer must be run *)
+    // remaining REVIEW:
```
Comment (KevinRansom) [inline src/absil/ilreflect.fs]: I have no idea what the intent of this comment is.

File: src/absil/ilreflect.fs [hunk 25]
```diff
@@ -1549,7 +1549,7 @@ let buildFieldPass2 cenv tref (typB:TypeBuilder) emEnv (fdef : ILFieldDef) =
         | Some initial -> 
             if not fieldT.IsEnum 
 #if FX_ATLEAST_45
-                || not fieldT.Assembly.IsDynamic // it is ok to init fields with type = enum  that are defined in other assemblies 
+                || not fieldT.Assembly.IsDynamic // it is ok to init fields with type = enum that are defined in other assemblies
```
Comment (KevinRansom) [inline src/absil/ilreflect.fs]: Move the comment to a separate line before the || not ....

File: src/absil/ilwrite.fs [hunk 26]
```diff
@@ -4212,7 +4212,7 @@ let writeBinaryAndReportMappings (outfile, ilg, pdbfile: string option, signer:
             else flags
           writeInt32AsUInt16 os dllCharacteristics
        // 000000e0
```
Comment (KevinRansom) [inline src/absil/ilwrite.fs]: Is this indented right?

File: src/fsharp/ErrorLogger.fs [hunk 27]
```diff
@@ -172,7 +172,7 @@ type PhasedError = { Exception:exn; Phase:BuildPhase } with
     /// Return true if the textual phase given is from the compile part of the build process.
     /// This set needs to be equal to the set of subcategories that the language service can produce. 
     static member IsSubcategoryOfCompile(subcategory:string) =
-        // Beware: This code logic is duplicated in DocumentTask.cs in the language service
+        // Beware: This code logic is duplicated in DocumentTask.cs in the language service.
```
Comment (KevinRansom) [inline src/fsharp/ErrorLogger.fs]: Beware: seems redundant and slightly dramatic.

File: src/fsharp/ErrorLogger.fs [hunk 28]
```diff
@@ -215,7 +215,7 @@ type PhasedError = { Exception:exn; Phase:BuildPhase } with
 [<System.Diagnostics.DebuggerDisplay("{DebugDisplay()}")>]
 type ErrorLogger(nameForDebugging:string) = 
     abstract ErrorCount: int
-    // the purpose of the 'Impl' factoring is so that you can put a breakpoint on the non-Impl code just below, and get a breakpoint for all implementations of error loggers
+    // The purpose of the 'Impl' factoring is so that you can put a breakpoint on the non-Impl code just below, and get a breakpoint for all implementations of error loggers
```
Comment (KevinRansom) [inline src/fsharp/ErrorLogger.fs]: The 'Impl' factoring enables a developer to place a breakpoint at the non-Impl code just below and get a breakpoint for all error logger implementations.

File: src/fsharp/ErrorLogger.fs [hunk 29]
```diff
@@ -273,7 +273,7 @@ module ErrorLoggerExtensions =
     open System.Reflection
 
     // Instruct the exception not to reset itself when thrown again.
-    // Why don?t we just not catch these in the first place? Because we made the design choice to ask the user to send mail to fsbugs@microsoft.com. 
+    // Why don't we just not catch these in the first place? Because we made the design choice to ask the user to send mail to fsbugs@microsoft.com.
```
Comment (KevinRansom) [inline src/fsharp/ErrorLogger.fs]: Design Note:      This enables the compiler to prompt the user to send mail to fsbugs@microsoft.com.

File: src/fsharp/ErrorLogger.fs [hunk 30]
```diff
@@ -273,7 +273,7 @@ module ErrorLoggerExtensions =
     open System.Reflection
 
     // Instruct the exception not to reset itself when thrown again.
-    // Why don?t we just not catch these in the first place? Because we made the design choice to ask the user to send mail to fsbugs@microsoft.com. 
+    // Why don't we just not catch these in the first place? Because we made the design choice to ask the user to send mail to fsbugs@microsoft.com. 
     // To achieve this, we need to catch the exception, report the email address and stack trace, and then reraise.
```
Comment (KevinRansom) [inline src/fsharp/ErrorLogger.fs]: by catching the exception, prompting and then propagating the exception with reraise.

File: src/fsharp/TraceCall.fs [hunk 31]
```diff
@@ -129,7 +129,7 @@ type internal Trace private() =
                 
     /// Log a method as its called.
     static member Call(loggingClass:string,functionName:string,descriptionFunc:unit->string) = Trace.CallImpl(loggingClass,functionName,descriptionFunc,None)
-    /// Log a method as its called. Expected always to be called on the same thread which will be named 'threadName'
+    /// Log a method as its called. Expected always to be called on the same thread which will be named 'threadName'.
```
Comment (KevinRansom) [inline src/fsharp/TraceCall.fs]: it's

File: src/fsharp/lib.fs [hunk 32]
```diff
@@ -392,8 +392,8 @@ let inline cacheOptRef cache f =
 // and goes depending on whether components are NGEN'd or not, e.g. 'ngen install FSharp.COmpiler.dll'
 // One workaround for the bug is to break NGEN loading and fixups into smaller fragments. Roughly speaking, the NGEN
 // loading process works by doing delayed fixups of references in NGEN code. This happens on a per-method
-// basis. For example, one manifestation is that a "print" before calling a method like LexFilter.create gets
-// displayed but the corresponding "print" in the body of that function doesn't get displayed. In between, the NGEN
+// basis. For example, one manifestation is that a 'print' before calling a method like LexFilter.create gets
```
Comment (KevinRansom) [inline src/fsharp/lib.fs]: e.g.

File: src/fsharp/lib.fs [hunk 33]
```diff
@@ -504,7 +504,7 @@ module internal AsyncUtil =
                 | Some res -> cont res)
                           
 
-        /// Get the result and commit it
+        /// Get the result and commit it.
```
Comment (KevinRansom) [inline src/fsharp/lib.fs]: Commit( ... )

File: src/utils/HashMultiMap.fs [hunk 34]
```diff
@@ -98,7 +98,7 @@ type internal HashMultiMap<'Key,'Value>(n: int, hasheq: IEqualityComparer<'Key>)
                     firstEntries.[y] <- h
                     rest.[y] <- t
                 | _ -> 
-                    // note: broken invariant
+                    // NOTE: broken invariant
```
Comment (KevinRansom) [inline src/utils/HashMultiMap.fs]: This comment seems a bit iffy.  If the invariant is broken ... swallowing it seems a poor choice.
Could you please review the code leading to this and make a proposal how to deal with it.

File: src/utils/HashMultiMap.fsi [hunk 35]
```diff
@@ -10,29 +10,29 @@ open System.Collections.Generic
...
-    /// Clear all elements from the collection
+    /// Clear all elements from the collection.
     member Clear   : unit    -> unit
     
-    /// Test if the collection contains any bindings for the given element
+    /// Test if the collection contains any bindings for the given element.
     member ContainsKey: 'Key -> bool
 
-    /// Remove the latest binding (if any) for the given element from the table
+    /// Remove the latest binding (if any) for the given element from the table.
```
Comment (KevinRansom) [inline src/utils/HashMultiMap.fsi]: if any does not need parens.

File: src/utils/TaggedCollections.fs [hunk 36]
```diff
@@ -711,7 +711,7 @@ namespace Internal.Utilities.Collections.Tagged
 #if ONE 
         | MapOne of 'Key * 'T
 #endif
-        // Note: performance rumour has it that the data held in this node should be
+        // NOTE: performance rumour has it that the data held in this node should be
```
Comment (KevinRansom) [inline src/utils/TaggedCollections.fs]: Please make an issue to review this and then label it ISSUE  nnn where nnn is the github issue ID.  Then someone can take a look and either remove the comment or make a change.

File: src/absil/il.fsi [hunk 37]
```diff
@@ -349,7 +349,7 @@ and
     | TypeVar of uint16           
     /// Custom modifiers. 
     | Modified of            
-          /// True if modifier is "required" 
+          /// <c>true</c> if modifier is "required".
```
Comment (dsyme) [inline src/absil/il.fsi]: We don't tend to use ``<c>`` markup in the F# compiler codebase.  I prefer the plain text here. thanks

File: src/absil/il.fsi [hunk 38]
```diff
@@ -1638,7 +1644,7 @@ val splitNamespace: string -> string list
 
 val splitNamespaceToArray: string -> string[]
 
-/// The splitILTypeName utility helps you split a string representing
+/// The <c>splitILTypeName</c> utility helps you split a string representing
```
Comment (dsyme) [inline src/absil/il.fsi]: FWIW I don't mind the use of ``<c>`` on true code identifiers.  I just found it a bit odd on ``true`` :)

File: src/absil/ilreflect.fs (hunk 24 above)
Comment (dsyme) [inline src/absil/ilreflect.fs]: It was written by the original developer listing the parts of the ILMethodDef data structure that have not yet been consumed by this function.  The parts are only used on mscorlib or in mixed-mode native/managed assemblies, or COM interop code, none of which is generated by ILReflect.  So the comment can be removed.

File: src/utils/HashMultiMap.fs (hunk 34 above)
Comment (dsyme) [inline src/utils/HashMultiMap.fs]: Just remove the comment. The code can't be reached but we're not about to add an exception to the codebase just for the fun of it, so let's deal with it by removing the comment.

File: src/utils/TaggedCollections.fs (hunk 36 above)
Comment (dsyme) [inline src/utils/TaggedCollections.fs]: Just remove the comment.  We could do work on immutable collections perf, but I don't think we should start with a random comment I wrote 8 years ago based on a corridor conversation :)  Systems.Collections.Immutable and the Roslyn immutable collections would be the place.

**Comment by WilliamBerryiii** [comment]
Thanks for the fast feedback.  I'll look across the review comments and take that in as I move forward.  Is the review chunk an acceptable size?

**Comment by WilliamBerryiii** [comment]
@KevinRansom how are things looking for this? Are you ready for this work to pick up again?

**Comment by WilliamBerryiii** [comment]
@KevinRansom I am going to break this up into a few smaller PRs - narrow the scope of the change to make it easier to work with.

**Comment by dsyme** [comment]
@WilliamBerryiii I'm not sure more PRs makes it easier to work with for @KevinRansom - though narrowing the scope of the change is good. AFAIK each PR has a more or less fixed overhead cost for integrating and releasing and Microsoft, ,but @KevinRansom can advise.

**Comment by KevinRansom** [comment]
@WilliamBerryiii  @dsyme The problem is not really the Numberof PR's it is the vast amount of churn the codebase has.  Any one of Don's PR's touches ton's of files.  And I have the coreclr branch which is also heavily edited.  I think this PR is just too soon.  I would like to wait until coreclr is merged into master before attempting this type of cleanup.  I'm just terrified of all of the possible conflicts.

Sorry, but I have to hold out a bit longer.

Kevin

**Comment by WilliamBerryiii** [comment]
Sounds great!  I'm ready to jump back in as soon as you give me the go-ahead.  If you want to house keep and close the PR(or have me do it) I have it backed up for when you are ready for it.

**Comment by dsyme** [comment]
@WilliamBerryiii If you could house keep (merge with master to remove the conflict) and keep it up-to-date that would be great.  As soon as there's a moment where there are no big feature PRs then I hope we can merge this

**Comment by dsyme** [comment]
Could someone resurrect this PR please?   Now would be a good time to merge it.,

**Comment by WilliamBerryiii** [comment]
I'll get to work on it.

**Comment by WilliamBerryiii** [comment]
@dsyme & @KevinRansom - I screwed up and no longer have the base repository for this PR.  Are you OK if I close this PR, and re-implement with all the above comments?

**Comment by dsyme** [comment]
@WilliamBerryiii I think you can recover the PR from the patch (see GitHub command line instructions above) and then do a forced push to a branch with the right name

File: src/absil/il.fs (hunk 3 above)
Comment (WilliamBerryiii) [inline src/absil/il.fs]: I believe this is a reference for the comment on line 3988 for the function 'computeCoveredBlocks'

File: src/absil/il.fs (hunk 1 above)
Comment (WilliamBerryiii) [inline src/absil/il.fs]: Fixed.

File: src/absil/il.fs (hunk 2 above)
Comment (WilliamBerryiii) [inline src/absil/il.fs]: Removed.

File: src/absil/il.fs (hunk 4 above)
Comment (WilliamBerryiii) [inline src/absil/il.fs]: Removed.

File: src/absil/il.fsi (hunk 5 above)
Comment (WilliamBerryiii) [inline src/absil/il.fsi]: Just remove the comment?

File: src/absil/il.fsi (hunk 6 above)
Comment (WilliamBerryiii) [inline src/absil/il.fsi]: Fixed.

File: src/absil/il.fsi (hunk 7 above)
Comment (WilliamBerryiii) [inline src/absil/il.fsi]: Issue 1066 created. Note added into file.

File: src/absil/il.fsi (hunk 9 above)
Comment (WilliamBerryiii) [inline src/absil/il.fsi]: Fixed.

File: src/absil/il.fsi (hunk 10 above)
Comment (WilliamBerryiii) [inline src/absil/il.fsi]: Fixed.

File: src/absil/il.fsi (hunk 11 above)
Comment (WilliamBerryiii) [inline src/absil/il.fsi]: Do you want this updated to say something else?

**Comment by KevinRansom** [comment]
Please do Bill … if anyone screwed up it was me … holding off for so long waiting on the coreclr branch.

Thanks for your contribution.

Kevin

File: src/absil/il.fsi (hunk 12 above)
Comment (WilliamBerryiii) [inline src/absil/il.fsi]: Updated.

File: src/absil/il.fsi (hunk 13 above)
Comment (WilliamBerryiii) [inline src/absil/il.fsi]: Updated.

File: src/absil/il.fsi (hunk 14 above)
Comment (WilliamBerryiii) [inline src/absil/il.fsi]: Fixed.

File: src/absil/illib.fs (hunk 15 above)
Comment (WilliamBerryiii) [inline src/absil/illib.fs]: Will fix.

File: src/absil/ilmorph.fs (hunk 16 above)
Comment (WilliamBerryiii) [inline src/absil/ilmorph.fs]: Removed.

File: src/absil/ilmorph.fsi (hunk 17 above)
Comment (WilliamBerryiii) [inline src/absil/ilmorph.fsi]: Removed.

File: src/absil/ilprint.fs (hunk 18 above)
Comment (WilliamBerryiii) [inline src/absil/ilprint.fs]: Updated.

File: src/absil/ilread.fs (hunk 19 above)
Comment (WilliamBerryiii) [inline src/absil/ilread.fs]: Updated.

File: src/absil/ilread.fs (hunk 20 above)
Comment (WilliamBerryiii) [inline src/absil/ilread.fs]: Corrected.

File: src/absil/ilreflect.fs (hunk 21 above)
Comment (WilliamBerryiii) [inline src/absil/ilreflect.fs]: Opened #1067

**Comment by WilliamBerryiii** [comment]
I was able to apply the patch and I am making the changes noted above.  Hopefully I can work some GIT love and get the updated changes into this PR.

File: src/absil/ilreflect.fs (hunk 23 above)
Comment (WilliamBerryiii) [inline src/absil/ilreflect.fs]: Fixed.

File: src/absil/ilreflect.fs (hunk 22 above)
Comment (WilliamBerryiii) [inline src/absil/ilreflect.fs]: Opened #1068

File: src/absil/ilreflect.fs (hunk 24 above)
Comment (WilliamBerryiii) [inline src/absil/ilreflect.fs]: Removed.

File: src/absil/ilreflect.fs (hunk 25 above)
Comment (WilliamBerryiii) [inline src/absil/ilreflect.fs]: Moved.

File: src/absil/ilwrite.fs (hunk 26 above)
Comment (WilliamBerryiii) [inline src/absil/ilwrite.fs]: It looks like this was done to make the sections easier to skip through visually.  The whole block of code probably needs some rework outside of this PR.

File: src/fsharp/ErrorLogger.fs (hunk 27 above)
Comment (WilliamBerryiii) [inline src/fsharp/ErrorLogger.fs]: Removed.

File: src/fsharp/ErrorLogger.fs (hunk 28 above)
Comment (WilliamBerryiii) [inline src/fsharp/ErrorLogger.fs]: Updated.

File: src/fsharp/ErrorLogger.fs (hunk 29 above)
Comment (WilliamBerryiii) [inline src/fsharp/ErrorLogger.fs]: Updated.

File: src/fsharp/ErrorLogger.fs (hunk 30 above)
Comment (WilliamBerryiii) [inline src/fsharp/ErrorLogger.fs]: Updated.

File: src/fsharp/TraceCall.fs (hunk 31 above)
Comment (WilliamBerryiii) [inline src/fsharp/TraceCall.fs]: Fixed Lines 130 and 132.

File: src/fsharp/lib.fs (hunk 32 above)
Comment (WilliamBerryiii) [inline src/fsharp/lib.fs]: Updated.

File: src/fsharp/lib.fs (hunk 33 above)
Comment (WilliamBerryiii) [inline src/fsharp/lib.fs]: Updated.

File: src/utils/HashMultiMap.fs (hunk 34 above)
Comment (WilliamBerryiii) [inline src/utils/HashMultiMap.fs]: Removed.

File: src/utils/HashMultiMap.fsi (hunk 35 above)
Comment (WilliamBerryiii) [inline src/utils/HashMultiMap.fsi]: Removed parens here and lines below.

File: src/utils/TaggedCollections.fs (hunk 36 above)
Comment (WilliamBerryiii) [inline src/utils/TaggedCollections.fs]: Removed.

File: src/absil/il.fsi (hunk 37 above)
Comment (WilliamBerryiii) [inline src/absil/il.fsi]: Reverted all occurrences.

**Comment by WilliamBerryiii** [comment]
@dsyme & @KevinRansom ... looks like according to this post I can't recover the old fork and fix this PR. https://github.com/isaacs/github/issues/168.  If it's OK with you, I'll close this with a note directing to the new Pull Request and visa versa.

**Comment by KevinRansom** [comment]
Sounds great.

**Comment by WilliamBerryiii** [comment]
Closing this PR as it has been replaced by #1069.

**Pull request closed without merging**